tests/commands/test_build.py

```diff
@@ -77,6 +78,16 @@ def basic_project(tmp_path):
     yield tmp_path
 
 
+@pytest.fixture(autouse=True)
+def set_message_handler_mode():
+    message_handler.mode = message_handler.NORMAL
+
+    def set_mode(mode):
+        message_handler.mode = mode
+
+    return set_mode
+
+
```

Comment:
Is this structure needed? Why not just doing below the following, instead of calling this?

```
monkeypatch.setattr(message_handler, 'mode', mode)
```

Reply:
That works, the fixture was more about also setting the default - i'll remove it though.

Reply:
Fixed, though had to update logsetup a bit and its tests to conform with using monkeypatch (being able to init to a level).